pandas/tests/io/formats/test_to_html.py

```diff
@@ -1868,12 +1868,12 @@ def test_to_html_no_index_max_rows(self):
     def test_to_html_notebook_has_style(self):
         df = pd.DataFrame({"A": [1, 2, 3]})
         result = df.to_html(notebook=True)
-        assert "thead tr:only-child" in result
+        assert "tbody tr th:only-of-type" in result
```

Comment:
can you add a few more assertions that lock down the style a bit more (maybe on the alignment).

Reply:
Can you update for this comment ?

Reply:
I added a couple more assertions a few days ago. Added `assert "vertical-align: middle;" not in result` and `assert "thead th" not in result`